tests/e2e/specs/upe-split/shopper/shopper-deferred-intent-creation-upe-enabled.spec.js

```diff
@@ -134,14 +143,43 @@ describe( 'Enabled UPE with deferred intent creation', () => {
 			await expect( page ).toMatch(
 				`${ card.expires.month }/${ card.expires.year }`
 			);
+			await waitTwentySecondsSinceLastCardAdded();
+		} );
+
+		it( 'should be able to set payment method as default', async () => {
+			await shopperWCP.goToPaymentMethods();
+			await shopperWCP.addNewPaymentMethod( 'basic2', card2 );
+			// Take note of the time when we added this card
+			timeAdded = Date.now();
+
+			// Verify that the card was added
+			await expect( page ).not.toMatch(
+				'You cannot add a new payment method so soon after the previous one. Please wait for 20 seconds.'
+			);
+			await expect( page ).toMatch( 'Payment method successfully added' );
+			await expect( page ).toMatch(
+				`${ card2.expires.month }/${ card2.expires.year }`
+			);
+			await shopperWCP.setDefaultPaymentMethod( card2.label );
+			// Verify that the card was set as default
+			await expect( page ).toMatch(
+				'This payment method was successfully set as your default.'
+			);
 		} );
 
-		it( 'should be able to delete the card', async () => {
+		it( 'should be able to delete cards', async () => {
 			await shopperWCP.deleteSavedPaymentMethod( card.label );
 			await expect( page ).toMatch( 'Payment method deleted.' );
+
+			await shopperWCP.deleteSavedPaymentMethod( card2.label );
+			await expect( page ).toMatch( 'Payment method deleted.' );
```

Comment:
delete card test is usually supposed to also clean up everything that was created during the test run. since we added two cards (two above tests) we need to clean both

Reply:
Perfect!